LICENSE

```diff
@@ -0,0 +1,428 @@
+Attribution-ShareAlike 4.0 International
```

Comment:
Why CC-BY-SA 4.0?

All of the STAC community extensions / specification are Apache 2.0. Anything we do here should be able to be freely incorporated into those efforts, even if not by us, which wouldn't be possible with an incompatible license.